utils/deploys/dependencies.ts

```diff
@@ -69,6 +69,10 @@ export default {
     1: "0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84",
     42: "0x4b7FCBC11BB45075b9A1F953128C09bC97D6a0D7",
   },
+  RETH: {
+    1: "0xae78736Cd615f374D3085123A210448E74Fc6393",
+    42: "",
```

Comment:
Only a few entries follow this pattern of defining a non existent deployment as empty string. With JS/TS being what it is... any reason why we couldn't just omit defining chain id 42 so that typing this data or checking for definition is more standard? Kovan is deprecated anyway so there will unlikely be a future deployment that we care about.

Reply:
Yeah will remove